app/components/files/files.tsx

```diff
@@ -73,7 +76,7 @@ const Files = ({canDownloadFiles, failed, filesInfo, isReplyPost, layoutWidth, l
 
     const renderItems = (items: FileInfo[], moreImagesCount = 0, includeGutter = false) => {
         let nonVisibleImagesCount: number;
-        let container: StyleProp<ViewStyle> = items.length > 1 ? styles.container : undefined;
+        let container: StyleProp<ViewStyle> = items.length > 1 ? styles.container : styles.marginTop;
```

Comment:
I don't get why this is needed, but seems to be old code, so I won't delay much on that.

Reply:
I'll make a post below explaining my reasoning.